src/pages/AddPage/AddPage.jsx

```diff
@@ -1,7 +1,46 @@
 import React from "react";
+import "./AddPage.scss";
+import { useState } from "react";
 
 const AddPage = () => {
-  return <h2>add</h2>;
+  const url = "http://localhost:3030";
+  const [text, setText] = useState("");
+  const [error, setError] = useState(null);
+  const [done, setDone] = useState(null);
+  const handleAddTodo = async () => {
+    await postTodo();
+  };
+
+  const postTodo = async () => {
+     try {
+      await fetch(`${url}/todos`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
```

Comment:
This header object can be a variable. Remember the naming from the previous todo project?
Name the variable and put it in `config.js` file and import it from there.
https://github.com/hedita/todo-react/pull/2/files#r1193175114

Reply:
Done.